README.md

```diff
@@ -1 +1,43 @@
-# setup-and-mobile-version-skeleton
\ No newline at end of file
+
+# Personal Portfolio:setup mobile skeleton
+
+> This is the mobile version of the personal portfolio website created with HTML5 and CSS. Html and Css linters have been added to help catch errors and make the coding very easy and bug free!!!
+
```

Comment:
- [ ] Kindly please include a screenshot of the project on your readme, and complete the other sections that were left blank.  This is a template, but you need to customize it for each project.  Please note this particular piece of feedback, because you will need it for all your future projects. ✔️ 

That way, your repo will look a lot more consistent and professional 💯

Reply:
Done.